backend/api/Services/RobotService.cs

```diff
@@ -52,6 +54,7 @@ public async Task<Robot> CreateFromQuery(CreateRobotQuery robotQuery)
                 _context.Entry(robotModel).State = EntityState.Unchanged;
                 await _context.Robots.AddAsync(newRobot);
                 await _context.SaveChangesAsync();
+                _ = _signalRService.SendMessageAsync("robot list updated", GetEnabledRobotsWithSubModels());
```

Comment:
```suggestion
                _ = _signalRService.SendMessageAsync("Robot updated", GetEnabledRobotsWithSubModels());
```

Should the event be named after what has happened? I assume the "robot list updated" indicates for the frontend that its list updated but what has happened is that one robot has changed.

Reply:
Should the names of these events be gathered in en enum like structure somewhere if they are important to be precise for the frontend?

Reply:
The reason I made it robot list updated is because it is a relative small and finite list. As a result it is cheap to update the whole list each time a change is detected. If we instead reacted to each update, creation and deletion we would require a lot more validation and temporary datastructure on the frontend. So I think this simplifies the code a lot without many obvious downsides (if needed we can always compare the lists to find what changed). The reason I did not do this for missionRuns is that there are too many of them and they are constantly increasing.